docker/docker-compose-2.10.yml

```diff
@@ -0,0 +1,125 @@
+version: "3"
+
+services:
+  ckan-2.10:
+    container_name: ckan-2.10
+    # image: ghcr.io/alphagov/ckan:2.10
```

Comment:
I've uncommented code lines that point to ghcr images but have left other local dev options commented so that a dev can see how to run images built locally.